test_crates/trait_unsafe_added/old/src/lib.rs

```diff
@@ -0,0 +1,14 @@
+// Public trait becomes unsafe, should get reported.
+pub trait TraitBecomesUnsafe {}
+
+// Trait that was not publicly-visible becomes unsafe, shouldn't get reported.
+trait TraitBecomesPublicAndUnsafe {}
+
+// Private trait becomes unsafe, shouldn't get reported.
+trait PrivateTraitBecomesUnsafe {}
```

Comment:
Consider adding the reverse test case too: a `pub trait` becomes both private and unsafe. It shouldn't get reported for being unsafe because the bigger problem is that it's now private.